code/style.css

```diff
@@ -1,4 +1,208 @@
-html {
-  background: #0025ff;
-  color: #fff;
+*, *::before, *::after {box-sizing: border-box;}
+
+body {
+  /***background: #a3a37d;***/
+  color: #727768;
+  font-family:'Open Sans Condensed', sans-serif;
+  margin: 0px;
 }
+
+/* HEADER+NAVBAR*/
+
+header {
+  background: white;
+  height: 100px;
+}
+
+header:hover {
+  background:#e5e6b4;
+  opacity: 0.5;
+  transition: all 500ms ease;
+  
+}
+
+nav a  {
+  text-decoration: none;
+  color: #727768;
+  margin: 10px;
+  padding: 10px 10px;
+  font-size: 20px;
+  float: right;
+}
+
+h1 {
+  font-size: 60px;
+}
+
+/* INLEDNING*/
+
+.big-news {
+  font-size: 20px;
+  margin-right: 20px;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  flex-wrap: wrap;
+  flex-direction: column;
+  width: 100%;
+  padding-top: 40px;
+  padding-bottom: 80px;
+
+}
+
+.big-news img {
+  width: 100%;
+  max-width: 100%;
+}
+
+.big-news div {
+  text-align: center;
+  padding: 25px;
+}
+
+/* CONTAINERS*/
+
+.plants {
+  background:  #d0d8c1;
+  padding: 25px;
+}
+.plants > h4 {
+  text-align: center;
+  margin: 70px 0px 70px 0px;
```

Comment:
once again, the shorthand for that can be `padding: 70px 0` (both top and bottom will be `70px` and right and left `0`)